### DISCUSSION TITLE
SONOFF THR320D Display and WEB UI do not show Energy Values

### DISCUSSION THREAD

**Discussion by DrMucki** [arendst/Tasmota#17970] [Q&A] [answered]
Hi out there.
I successfully flashed tasmonta32.bin (Version 12.4.0) on my Sonor TRHR320D Elite (the one with a display on it)

I use this Template from https://templates.blakadder.com/sonoff_THR320D.html
{"NAME":"Sonoff THR320D","GPIO":[32,0,0,0,226,9280,0,0,0,321,0,576,320,9184,9216,9312,0,0,9313,9248,0,1,0,3840,0,0,0,0,0,0,0,0,0,0,0,0],"FLAG":0,"BASE":1}

and activated it.

I can switch toggle 1 and toggle 2 , and a 230V Bulb can be switched by toggle 1 and the hardware button.
But there are no Energy meters shown on the Web UI and the display remains dark.... At Reboot it shows ----------

I also tried the Auto Configuration Options, did a factory reset on the settings, refreshed the firmware

GPIO 25 is set to user and can be configured to something. Now it is "none". I tried a few with TM1621 but nothing worked for me..

any help is appreciated

**Comment by sfromis** [comment, marked as answer]
That device does not have the feature of energy monitoring. What you can do is to connect a temperature probe, autoconfig for the particular probe type (this will configure gpio 25), and expect to get temperature (and possibly humidity) on the display.
